vaibhav/cart/shopping_cart.rb

```diff
@@ -0,0 +1,128 @@
+#shopping cart 
+
+class Shopping_cart
+
+    def initialize
+        @ary = Array.new{{}}
+        @cart = Array.new{{}}
+    end
+
+    def Add_Products(ary)
+        print 'Enter product name: '
+        productName = gets
```

Comment:
you can use gets.chomp.lowcase.strip for better input

Reply:
this file is for testing purpose.